README.md

```diff
@@ -610,6 +610,31 @@ Submit comments or suggestions by opening an [issue](https://github.com/vegastri
 
 And if you can design some missions it would rock!
 
+Vega Strike Contacts
+====================
+
+Vega Strike is the product of many contributors from all around the world. If you need help, find a bug, want to request a features, etc then please contact us all using one
```

Comment:
Should be `feature` rather than `features`

Reply:
fixed, thanks for catching that!